code/game/machinery/smelter.dm

```diff
@@ -128,6 +127,20 @@
 	for(var/obj/O in smelting.contents)
 		smelt_item(O)
 
+	//Occulus Edit Start: Smelting Mobs
+	for(var/mob/oopsalltori in smelting.contents)
```

Comment:
*scream

Reply:
How does it work with roaches and simplemobs and stuff if it doesn't sanitize down to the human type tho?

Reply:
take_overall_damage is a proc on mob/living
the organ separating proc only checks contents for external organs
the drop_all_from_inventory is only called if an item is in the contents of a mob and is flagged as equipped.

Reply:
Also I did test it with roaches and simplemobs. Most of them gibbed from the high damage, but Fuhrer's survive the process.